doc/Overview/Http/HttpOverview.md

```diff
@@ -3,17 +3,35 @@ uid: Overview.Http
 ---
 # HTTP
 
-Uno.Extensions.Http allows for the registration of API endpoints as multiple typed `HttpClient` instances. In this centralized location for accessing web resources, the lifecycle of the corresponding `HttpMessageHandler` objects is managed. Added clients can optionally be configured to use the platform-native handler. Additional functionality is provided to clear cookies or log diagnostic messages in responses. This library uses [Microsoft.Extensions.Http](https://www.nuget.org/packages/Microsoft.Extensions.Http) for any HTTP related work.
+Uno.Extensions.Http allows for the registration of API **endpoints** as multiple typed `HttpClient` instances. In this centralized location for accessing web resources, the lifecycle of the corresponding `HttpMessageHandler` objects is managed. Added clients can optionally be configured to use the platform-native handler. Additional functionality is provided to clear cookies or log diagnostic messages in responses. This library uses [Microsoft.Extensions.Http](https://www.nuget.org/packages/Microsoft.Extensions.Http) for any HTTP related work.
 
-For more documentation on HTTP requests, read the references listed at the bottom.
+For additional documentation on HTTP requests, read the references listed at the bottom.
 
 ## Register Endpoints
 
 Web resources exposed through an API are defined in the application as clients. These client registrations include type arguments and endpoints to be used for the client. The endpoint is defined in the `EndpointOptions` class. While it uses the platform-native HTTP handler by default, this value can be configured. 
 
 ```csharp
-private IHost Host { get; }
+protected override void OnLaunched(LaunchActivatedEventArgs args)
+{
+    var builder = this.CreateBuilder(args)
+        .Configure(host => {
+            host
+            .UseHttp((context, services) =>
+            {
+                services
+                .AddClient<IShowService, ShowService>(context, "configsectionname");
+            });
+        });
+...
+```
+
+> [!TIP]
+> If configuration sections are already used elsewhere, continuing to use that approach offers uniformity and broader accessibility of endpoint options. Consider whether this type of access is needed before using the alternate method below.
```

Comment:
@nickrandolph Does this say enough to convey our recommendation (or lack thereof)?